joblib/test/test_func_inspect.py

```diff
@@ -207,13 +213,13 @@ def test_clean_win_chars():
         assert char not in mangled_string
 
 
-def test_format_signature():
+@parametrize(['func', 'args', 'kwargs', 'sgn_expected'],
+             [(g, [list(range(5))], {}, 'g([0, 1, 2, 3, 4])'),
+              (k, [1, 2, (3, 4)], {'y': True}, 'k(1, 2, (3, 4), y=True)')])
```

Comment:
It was just to keep things within 80 characters. I would remember about the fact, keeping in mind that there might be some piece of code which would be written a long time ago due to some reason. :smile: